docs/reference/analysis/tokenfilters/length-tokenfilter.asciidoc

```diff
@@ -11,6 +11,6 @@ type:
 |===========================================================
 |Setting |Description
 |`min` |The minimum number. Defaults to `0`.
-|`max` |The maximum number. Defaults to `Integer.MAX_VALUE`.
+|`max` |The maximum number. Defaults to `Integer.MAX_VALUE`, which is `2^32-1` or 4294967295.
```

Comment:
Actually, I was a bit to hasty, this should be `2^31-1` or 2147483647. Since I already merged the first change I'm going to push a fix afterwards, will squash this for the upstream branches. Sorry I missed this first.